deploy/scripts/src/cri-orange/kbv.ts

```diff
@@ -106,13 +114,13 @@ export function kbv(): void {
     }
   )
 
-  sleepBetween(1, 3)
+  sleepBetween(3, 5)
 
   // B01_KBV_04_KBVQuestion3
   timeGroup(groups[3], () => {
     // 01_KBVCRICall
     res = timeGroup(
-      groups[4].split('::')[1],
+      groups[6].split('::')[3],
```

Comment:
```suggestion
      groups[6].split('::')[1],
```
The second index here should not be changed